src/less/expressive-loader/expressive-loader.less

```diff
@@ -1,10 +1,24 @@
 @import "../variables/variables.less";
 
 :root {
-    --expressive-loader-message-fadein-duration: 833ms;
-    --expressive-loader-message-fadeout-duration: 400ms;
+    --expressive-loader-message-animatein-duration: 833ms;
+    --expressive-loader-message-animateout-duration: 400ms;
     --expressive-loader-scroll-lines-duration: 20s;
     --expressive-loader-scale-line-duration: 0.7s;
+    --expressive-loader-line-height: 6px;
+    --expressive-loader-line-gap: var(--spacing-50);
+    --expressive-loader-line-border-radius: 2px;
+    --expressive-loader-color-1: #e53238;
```

Comment:
Moved color variables into this file, rather than the overarching dark/light theme files. I'm leaning towards using color-{number} instead of color-{colorName} to make it more flexible. Thoughts?